gateway/dataimpl/server_v1/bucketadminserver.go

```diff
@@ -202,7 +206,7 @@ func (s *BucketAdminServer) CreateBucket(
 				DurabilityMinLevel: minimumDurabilityLevel,
 			},
 			ConflictResolutionType: conflictResolutionType,
-			ReplicaIndexDisabled:   !replicaIndexes,
+			ReplicaIndex:           replicaIndexes,
```

Comment:
nit: the `ReplicaIndexes` and `ReplicaIndex` variable name interchange is a bit confusing

Reply:
I think it's actually named wrong in gocbcorex.